src/clr-angular/utils/smart-popover/smart-popover-content.ts

```diff
@@ -84,12 +84,6 @@ export class ClrSmartPopoverContent implements AfterContentChecked, OnDestroy {
     this.view.rootNodes.forEach(node => this.renderer.removeChild(this.document.body, node));
     this.container.clear();
     delete this.view;
-
-    // Clean up for when popovers do have to go off screen
-    this.renderer.removeStyle(this.document.body, 'position');
-    this.renderer.removeStyle(this.document.body, 'width');
-    this.renderer.removeStyle(this.document.body, 'height');
```

Comment:
I'm happy that we can go without these. Glad we don't need to change anything in the global scope.